content/en/docs/releases/kubeflow-1.10.md

```diff
@@ -0,0 +1,295 @@
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/notebook-controller">v1.10.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Profile Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/profile-controller">v1.10.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/notebook-controller">v1.10.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/crud-web-apps/volumes">v1.10.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Volumes Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/crud-web-apps/tensorboards">v1.10.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Pipelines Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Pipelines Working Group</td>
+        <td>Kubeflow Pipelines</td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines/releases/tag/2.4.1">v2.4.1</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Serving Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Serving Working Group</td>
+        <td>KServe</td>
+        <td>
+          <a href="https://github.com/kserve/kserve/releases/tag/v0.14.1">v0.14.1</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Training Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Training Working Group</td>
+        <td>Training Operator</td>
+        <td>
+          <a href="https://github.com/kubeflow/training-operator/releases/tag/v1.9.0">v1.9.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Data Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="2" class="align-middle">Data Working Group</td>
+        <td>Model Registry</td>
+        <td>
+          <a href="https://github.com/kubeflow/model-registry/releases/tag/v0.2.15">v0.2.15</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Spark Operator</td>
+        <td>
+          <a href="https://github.com/kubeflow/spark-operator/releases/tag/v2.1.0">v2.1.0</a>
+        </td>
+      </tr>
+  </tbody>
+</table>
+</div>
+
+### Dependency Versions (Manifests)
+
+{{% alert title="Note" color="warning" %}}
+This information is only for the manifests found in the <a href="https://github.com/kubeflow/manifests">kubeflow/manifests</a> repository, packaged distributions may have different requirements or supported versions.
+{{% /alert %}}
```

Comment:
I would like to remove this alert, otherwise  lgtm

Reply:
I think it's good to have this alert as we don't have control over Distribution manifests

Reply:
It is very important, as it's very common for distributions to have different requirements and supportred versions.